BoneReconstructionPlanner/BoneReconstructionPlanner.py

```diff
@@ -1459,14 +1522,19 @@ def createMiterBox(self, X, Y, Z, name):
 
 
   def createFiducialList(self):
+    shNode = slicer.mrmlScene.GetSubjectHierarchyNode()
+    cylindersFiducialsListsFolder = shNode.GetItemByName("Cylinders Fiducials Lists")
+    if cylindersFiducialsListsFolder == 0:

```

Comment:
I use the folder existence to detect if cylinders fiducials were already created to enable "Create cylinders from fiducial list and surgical guide base" button. What should I do instead?

Reply:
My comment was not for the functionality, just style: `not x` is more Pythonic (and somewhat shorter, simpler) than `x==0`.

```python
if not cylindersFiducialsListsFolder:
```

instead of the current:

```python
if cylindersFiducialsListsFolder == 0:
```